client/reader/stream/index.jsx

```diff
@@ -419,8 +421,15 @@ export default class ReaderStream extends React.Component {
 		const count = this.state.posts.length ? 2 : this.props.store.getPerPage(),
 			placeholders = [];
 
-		times( count, function( i ) {
-			placeholders.push( <PostPlaceholder key={ 'feed-post-placeholder-' + i } /> );
+		times( count, ( i ) => {
```

Comment:
as long as we're here...

`times` returns an array of every item returned from the callback. So if we return the Placeholder, we can just make this `return times( count, ... )` and drop the `placeholders` variable entirely.